src/applications/personalization/profile-2/manifest.json

```diff
@@ -7,13 +7,6 @@
   "template": {
     "vagovprod": false,
     "title": "Your Profile",
-    "layout": "page-react.html",
-    "includeBreadcrumbs": true,
-    "breadcrumbs_override": [
-      {
-        "path": "profile-2/",
-        "name": "Your profile"
-      }
-    ]
+    "layout": "page-react.html"
   }
-}
+}
```

Comment:
weird that your editor seems to strip newlines from the end of .json files but not other files?